### PR TITLE
add docker_base_name parameter to build_vc_docker.sh 

### PR DESCRIPTION
**Pull request by TammushB**

### CODE REVIEW

**Review by ilyasoifer — COMMENTED** [review]
can we change jukebox_vc to ugvc ?

File: build_vc_docker.sh [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-DOCKER_BASE_NAME="gcr.io/terra-project-249020/jukebox_vc:"
+DOCKER_BASE_NAME="gcr.io/ultima-data-307918/jukebox_vc:"
 
 function usage () {
   echo -e "
   ./build_vc_image.sh -v <git_tag_or_hash> -t <docker_tag> (-h | --help)
 
   -v | --version    git tag or hash to checkout VariantCalling
   -t | --tag        tag with the version number for resulting docker (e.g. 1.1)
+  -base | --docker_base_name docker base name (default is : \"gcr.io/ultima-data-307918/jukebox_vc:\" )
```
Comment (giuliovn) [inline build_vc_docker.sh:11]: you can use ${DOCKER_BASE_NAME} to use the default value denied in the variable

**Review by giuliovn — APPROVED** [review]
I also pushed a correction to the messages of the old flags

File: build_vc_docker.sh (hunk 1 above)
Comment (TammushB) [reply build_vc_docker.sh]: Correct.

**Comment by TammushB** [comment]
> can we change jukebox_vc to ugvc ?

Done. When and how do we update the template files to use the correct docker image reference ?

**Comment by TammushB** [comment]
Merging this

**Review by ilyasoifer — APPROVED** [review]

**Pull request merged**